test/integration/consul-container/test/basic/connect_service_test.go

```diff
@@ -22,7 +22,14 @@ import (
 //   - Make sure a call to the client sidecar local bind port returns a response from the upstream, static-server
 func TestBasicConnectService(t *testing.T) {
 	t.Parallel()
-	cluster := createCluster(t)
+
+	buildOptions := &libcluster.BuildOptions{
+		InjectAutoEncryption:   true,
+		InjectGossipEncryption: true,
+		// TODO: fix the test to not need the service/envoy stack to use :8500
```

Comment:
Is this a future TODO or should it be resolved in this PR?

Reply:
Future todo. I tried to make sense of it in a timebox and didn't get anywhere. Amended todo with creator's initials.